src/sentry/integrations/jira/integration.py

```diff
@@ -541,19 +541,38 @@ def fetch_issue_create_meta(self, client, project_id):
             )
         return meta
 
-    def get_create_issue_config_no_params(self):
-        return self.get_create_issue_config(None, None, params={})
-
     def get_create_issue_config(self, group, user, **kwargs):
+        """
+        We use the `group` to get three things: organization_slug, project
+        defaults, and default title and description. In the case where we're
+        getting `createIssueConfig` from Jira for Ticket Rules, we don't know
+        the issue group beforehand.
+
+        :param group: (Optional) Group model.
+        :param user: User model. TODO Make this the first parameter.
+        :param kwargs: (Optional) Object
+            * params: (Optional) Object
+            * params.project: (Optional) String. The Jira project ID.
+            * params.issuetype: (Optional) String. The Jira issue type. For
+                example: "Bug", "Epic", "Story".
+        :return:
+        """
+        kwargs = kwargs or {}
         kwargs["link_referrer"] = "jira_integration"
         params = kwargs.get("params", {})
         fields = []
         defaults = {}
+
+        project_param_option = params.get("project")
+        project_option = None
         if group:
             fields = super(JiraIntegration, self).get_create_issue_config(group, user, **kwargs)
-            defaults = self.get_defaults(group.project, user)
+            project_option = group.project
+        project_option = project_param_option or project_option
```

Comment:
@mgaeta I am a bit confused if `project_option` is the Jira project or the Sentry project. The doc string says that `project` is the Jira project but `group.project` is the Sentry project.

Reply:
You're right this didn't work and I reverted.